adaptor/adaptor.go

```diff
@@ -126,8 +146,8 @@ func (c *Config) splitNamespace() (string, string, error) {
 }
 
 // CompileNamespace split's on the first '.' and then compiles the second portion to use as the msg filter
```

Comment:
with `V2` adaptors, `Config` never becomes a concrete object so these are now package level methods. I'm still planning on a near future change to remove the "db" part of this such that the only thing that is in `namespace` is the regexp.